amr.md

```diff
@@ -2700,36 +2700,33 @@ Annotators are encouraged to use `have-degree-91` as the root concept (as oppose
 > That boy is more bright.
 
 ```lisp
-(b / bright-03
-   :ARG1 (b2 / boy
-               :mod (t / that))
-   :degree (m / most))
+(b / boy 
+      :ARG1-of (h / have-degree-91 
```

Comment:
Why does this example use `have-degree-91` where "The boy is more bright" does not?

Reply:
Apologies, they should both use have-degree-91 under Ulf's proposal -- I suppose it is an implicit comparison.